src/libs/actions/BankAccounts.js

```diff
@@ -751,7 +753,8 @@ function setupWithdrawalAccount(data) {
                 }
             } else {
                 if (response.jsonCode === 666 || response.jsonCode === 404) {
-                    error = response.message;
+                    error = response.message || response.htmlMessage;
```

Comment:
I am not sure I understand this change? In general we prefer the HTML message first (on OldDot)